RecoTracker/TkSeedGenerator/plugins/BuildFile.xml

```diff
@@ -1,3 +1,5 @@
+<use name="PhysicsTools/TensorFlow"/>
+<use name="SimDataFormats/TrackingAnalysis"/>
```

Comment:
I would tend to agree that you don't want to add a Sim dependency here. Hence the JetCoreMCtruthSeedGenerator should be moved elsewhere, e.g. validation

Reply:
moved to validation